services/CertificateGeneration/DockerFile

```diff
@@ -0,0 +1,21 @@
+FROM microsoft/dotnet:2.0-sdk AS build
+WORKDIR /app
+
+COPY *.sln .
+COPY CertificateGeneration.NetCore/*.csproj ./CertificateGeneration.NetCore/
+COPY CertificateGenerationTests.NetCore/*.csproj ./CertificateGenerationTests.NetCore/

```

Comment:
Do you need the tests in the docker image?

Reply:
Yes for build image. Docker runs unittests (see line 16-18), if they fail docker build fails.